packages/marshal/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@agoric/marshal",
   "version": "0.2.7-dev.0",
   "description": "marshal",
-  "main": "marshal.js",
+  "main": "index.js",
```

Comment:
Please add `"*.js"` to the `"files"` list.

Reply:
Which "files" list? The only I see in package.json is for ava at
https://github.com/Agoric/agoric-sdk/blob/af0a2b041744a75f634936ab50c1be28ec3dbf32/packages/marshal/package.json#L96-L99
Adding "*.js" there doesn't seem right and doesn't work anyway.

Reply:
After seeing same-structure/package.json 's "files", I found an explanation at https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files and added one here as
```js
  "files": [
    "src/",
    "*.js",
    "NEWS.md"
  ],
```